xbmc/PartyModeManager.h

```diff
@@ -80,8 +74,7 @@ class CPartyModeManager final
   int m_iRandomSongs;
 
   // history
-  unsigned int m_songsInHistory;
-  std::vector< std::pair<int,int> > m_history;
+  std::vector< std::pair<int, int> > songIDCache;
```

Comment:
```suggestion
  std::vector< std::pair<int, int>> m_songIDCache;
```

Code style: members must be prefixed `m_`

Reply:
Overlooked extra space, make that `std::vector<std::pair<int, int>> m_songIDCache;`